promise-it-wont-hurt-solutions/src/11-multiple-promises.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function all(promise1, promise2) {
+  return new Promise(function (fulfill, reject) {
+    var counter = 0;
+    var arr = [];
+
+    promise1.then(function (count) {
+      arr[0] = count;
+      counter++;
+
+      if (counter === 2) {
```

Comment:
a ternary operator would be good economy of code 
counter === 2 ? fulfill(arr) : reject('some reject message')